Dockerfile-app

```diff
@@ -16,6 +16,9 @@ RUN yarn install --frozen-lockfile \
     && yarn cache clean \
     && rm -rf /npm-packages-offline-cache
 
+# Do we want to run the compile step here? Or elsewhere in the pipeline?
+RUN yarn compile
```

Comment:
Just thinking about this - shouldn't this be moved until _after_ everything is copied in ?